src/Generator.h

```diff
@@ -1257,9 +1257,9 @@ class StubInput {
 public:
     // *not* explicit.
     template<typename T2>
-    StubInput(const StubInputBuffer<T2> &b) : kind_(IOKind::Buffer), parameter_(b.parameter_) {}
-    StubInput(const Func &f) : kind_(IOKind::Function), parameter_(), func_(f) {}
-    StubInput(const Expr &e) : kind_(IOKind::Scalar), parameter_(), expr_(e) {}
+      StubInput(const StubInputBuffer<T2> &b) : kind_(IOKind::Buffer), parameter_(b.parameter_), func_(), expr_() {}
```

Comment:
Why are these changes necessary?

Reply:
Sorry, I think I had needed this when i was using an older compiler. Will push an update